### ISSUE TITLE
PLARD training

### ISSUE THREAD

**Issue by go925315** [zhechen/PLARD#7]
Hi,

Thank you very much for your code. 

I use the pspnet101_cityscapes pre-trained model to train kitti, but I cannot achieve the results of the paper. Can you provide suggestions?

dataset: kitti training 289 sheets
model: PLARD (no modification)
batch_size: 8
Learning Rate: 1e-4 -> 1e-6 (SGD)
epoch: 80

![um_road_000000](https://user-images.githubusercontent.com/14261085/100409419-bf78c700-30a6-11eb-9632-02a426a554ec.png)

Many thanks

**Comment by go925315** [comment]
Thank you author, the problem is solved.

**Comment by OSU-Haolin** [comment]
> Thank you author, the problem is solved.

could you tell me how to solve it? I meet the same problem.

**Comment by mei123hao** [comment]
> Thank you author, the problem is solved.

could you tell me how to solve it? I meet the same problem.

**Comment by whz-520** [comment]
你能告诉我怎么解决吗？我遇到了同样的问题。
